astroquery/exceptions.py

```diff
@@ -111,3 +112,11 @@ class EmptyResponseError(ValueError):
     Astroquery error class to be raised when the query returns an empty result
     """
     pass
+
+
+class BadRowWarning(AstropyWarning):
```

Comment:
I'm not so sure about the naming of this, is this a row, or an entry, or an object? after all this warning is expected with e.g. vectorized `query_object` queries. (what is actually/orginal error/warning coming back? (I suppose I can check on it tomorrow myself)

Reply:
The error can vary.  For example, in one of the tests, it is:
```
[5] 'gHer' this identifier has an incorrect format for catalogs:
	gH : Gurwell + Hodge
	g : Giclas
```
In another case, it is:
```
[5] 'dosntexist': No known catalog could be found
```

But the net effect is to not include that search term in the output results.

Maybe this should be `MissingRowWarning` or `BlankResponseWarning`?  I think it is right that we raise a warning rather than an error, because it is often valuable to run a query where you do not expect results for all search terms.

Reply:
Still, my problem with the "Row" remains, it's feels like not the right word. MissingObjectWarning/MissingTargetWarning/BadObjectWarning, or in fact your suggestion of BlankResponseWarning could all work better.

(it's an implementation detail that the result will be returned one object per row, it could be a different structure, too, so I think the warning should really refer to this detail)

Reply:
Yeah sounds good - I'll change to... I guess `BlankResponseWarning`, then?  It's not necessarily a missing object, since the first example says 'there might be an object there but you got the name format wrong'.